### PR TITLE
chore: update config bootstrapper

### PR DESCRIPTION
**Pull request by aaron-steinfeld**
## Description
Update to newer config bootstrapper which includes newer entity clients to support bootstrapping latest features.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (aaron-steinfeld) [inline .github/CODEOWNERS:4]: Updated since I was in here

**Review by tim-mwangi — APPROVED** [review]

File: .github/CODEOWNERS
Comment (aaron-steinfeld) [reply .github/CODEOWNERS]: Nevermind, reverted to unblock this change since it's a holiday in India and I'm the only code owner not based in India :) Pulled into a separate PR: https://github.com/hypertrace/entity-service/pull/101

**Review by tim-mwangi — APPROVED** [review]

**Pull request merged**
